lightning-persister/src/lib.rs

```diff
@@ -143,7 +143,7 @@ impl FilesystemPersister {
 		}
 }
 
-impl<ChannelSigner: Sign + Send + Sync> channelmonitor::Persist<ChannelSigner> for FilesystemPersister {
+impl<ChannelSigner: Sign+Clone> channelmonitor::Persist<ChannelSigner> for FilesystemPersister {
```

Comment:
I think you can entirely drop the last commit - the prev commit already removed the `send + Sync` requirement and the last commit just re-adds Clone which is already implied.

Reply:
Rebase error.  Done.